src/assets/logo.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="405" height="244" viewBox="0 0 405 244" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
This is actually an SVG file with an embedded PNG file. We should avoid that due to potential issues and also keep the text separate to prevent breakage (screenreaders, image compression, etc)